project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
   val nailgunVersion = "ee3c4343"
   // Used to download the python client instead of resolving
   val nailgunCommit = "a2520c1e"
-  val zincVersion = "1.3.0-M4+45-d4354be3"
+  val zincVersion = "1.3.0-M4+46-edbe573e"
```

Comment:
The tests seem no to be able to fetch this version I think? Is it published or maybe not published yet?

Reply:
I published it just a few minutes before opening this PR. I'll restart the build, as I think it's working fine on my machine 🤔